.circleci/config.yml

```diff
@@ -55,6 +55,11 @@ commands:
           name: Notify Slack on failure
           command: ./.circleci/slack-notify-failure.sh "${CIRCLE_JOB}"
           when: on_fail
+  notify_slack_on_success:
+    steps:
+      - run:
+          name: Notify Slack on success
+          command: ./.circleci/slack-notify-success.sh
```

Comment:
does it need any ```when:``` clause or we're good here?

Reply:
the default `when:` is `on_success` :)